src/database/types.ts

```diff
@@ -3,7 +3,7 @@
  * Please do not edit it manually.
  */
 
-import type { ColumnType } from "kysely";
+import type { ColumnType } from "kysely"
```

Comment:
only thing with this and removing the reference in `eslint.config.mjs` is that next time the types are re-generated (ie after some migration) it will put the `;` back in automatically. ideally there's a way to use the kysely ctl library to control that when the migrations run and the `types` file is generated, however i couldn't find anything. if we do want to be manually removing the `;` from the file each time a migration is ran, we will probably want to remove the entry in `prettier.ignore` as well cause that is currently ignoring this file.

overall technically we shouldn't be "editing" this file (even though its not an edit of any substance), but just throwing this out there as to why it was originally included in the eslint/prettier ignores

Reply:
Yes, it does put the `;` back (as well as a bunch of other changes),  but we then re-run eslint/prettier to auto-fix the files right afterwards in ./dev/migrate-dev-db